Stellar-transaction.x

```diff
@@ -695,6 +707,8 @@ struct Operation
         ExtendFootprintTTLOp extendFootprintTTLOp;
     case RESTORE_FOOTPRINT:
         RestoreFootprintOp restoreFootprintOp;
+    case PAD_FOR_TESTING:
+        PadForTestingOp padForTestingOp;
```

Comment:
Is there a way to be able to do this without introducing a new operation? Maybe a development feature of xdrpp to be able to specify during codegen a union type that gets additional padding arm for testing?

Adding it to the XDR will cause all downstream systems and integrators to see `PadForTesting` as a valid Stellar operation they need to handle when decoding operations (e.g. Rust decoders will be forced to handle the arm), or display in UIs of development tools, and will be included in things like the JSON schema, etc.

Reply:
Thanks for flagging this: I'll investigate alternative solutions.